Example/MBViewController.m

```diff
@@ -114,7 +114,7 @@ - (void)startNavigation:(MBRoute *)route {
     MBNavigationService *service = [[MBNavigationService alloc] initWithRoute:route directions:self.directions locationSource:nil eventsManagerType:nil simulating:MBNavigationSimulationOptionsAlways routerType:nil];
     
     self.navigation = service;
-    MBNavigationViewController *controller = [[MBNavigationViewController alloc] initWithRoute:route styles:nil navigationService:service voiceController: nil];
+    MBNavigationViewController *controller = [[MBNavigationViewController alloc] initWithRoute:route styles:nil navigationService:service voiceController: nil bottomBanner:nil];
```

Comment:
Should we consider making this new param more generic? Possibly an array of banners? (...considering banner could be a top or bottom banner.)